src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+
+    private HashUtil() {
+
```

Comment:
```suggestion
```

Reply:
'CTOR_DEF' should be separated from previous statement. [EmptyLineSeparator]

Reply:
weird, but ok